.github/workflows/pkgci_regression_test.yml

```diff
@@ -254,10 +260,49 @@ jobs:
             --log-cli-level=info \
             --timeout=1200 \
             --durations=0 \
-            --config-files=${SDXL_CONFIG_FILE_PATH}
+            --config-files=${SDXL_UNET_CONFIG_FILE_PATH}
+
+      - name: "Run external tests - SDXL prompt encoder"
+        if: "matrix.sdxl-clip-config-file != '' && !cancelled()"
+        run: |
+          source ${VENV_DIR}/bin/activate
+          pytest SHARK-TestSuite/iree_tests/pytorch/models/sdxl-prompt-encoder-tank \
+            -rpfE \
+            -k real_weights \
+            --no-skip-tests-missing-files \
+            --capture=no \
+            --log-cli-level=info \
+            --timeout=1200 \
+            --durations=0 \
+            --config-files=${SDXL_CLIP_CONFIG_FILE_PATH}
+
+      - name: "Run external tests - SDXL vae decode"
+        if: "matrix.sdxl-vae-config-file != '' && !cancelled()"
+        run: |
+          source ${VENV_DIR}/bin/activate
+          pytest SHARK-TestSuite/iree_tests/pytorch/models/sdxl-vae-decode-tank \
+            -rpfE \
+            -k real_weights \
+            --no-skip-tests-missing-files \
+            --capture=no \
+            --log-cli-level=info \
+            --timeout=1200 \
+            --durations=0 \
+            --config-files=${SDXL_VAE_CONFIG_FILE_PATH}
 
       - name: "Running SDXL rocm pipeline benchmark"
         if: contains(matrix.name, 'rocm')
         run: |
           source ${VENV_DIR}/bin/activate
-          bash SHARK-TestSuite/iree_tests/benchmarks/benchmark_sdxl_rocm.sh
+          pytest SHARK-TestSuite/iree_tests/benchmarks/benchmark_sdxl_rocm.py \
+            --goldentime-rocm-e2e-ms 1636 \
+            --goldentime-rocm-unet-ms 442 \
+            --goldentime-rocm-clip-ms 16.5 \
+            --goldentime-rocm-vae-ms 285 \
+            --gpu-number 3 \
+            --rocm-chip gfx90a \
+            --log-cli-level=info \
+            --retries 7
```

Comment:
oof, 7 retries x_x

that seems fairly arbitrary, so I expect we'll be tuning the thresholds, benchmark iterations, and retry count ...

Reply:
Yup, that was for when I temporarily changed the unet to 441. Thought it wouldn't hurt to leave it at 7, but in the testing I did, never got there with these values. But, yeah we can tune these numbers as we go :)